### PR TITLE
Add "code for transparent background image" to article

### PR DESCRIPTION
**Pull request by PanSem**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/css/tutorials/transparent-background-images/index.md [hunk 1]
```diff
@@ -3,13 +3,37 @@ title: Transparent Background Images
...
+  height: 100px;
+  top: 0;
+  left: 0;
+  position: absolute;
+  z-index: -10;
+  content: "";
+}
+```
+
+You must have same width and height for div and div:before
```
Comment (co16353sidak) [inline guide/english/css/tutorials/transparent-background-images/index.md]: `have same` -> `have the same`

File: guide/english/css/tutorials/transparent-background-images/index.md [hunk 2]
```diff
@@ -3,13 +3,37 @@ title: Transparent Background Images
 ---
 ## Transparent Background Images
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/tutorials/transparent-background-images/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+You have to add :before to the element which you want to add the transparent background image
```
Comment (co16353sidak) [inline guide/english/css/tutorials/transparent-background-images/index.md]: `to the element which` -> `to the element to which`

Comment (co16353sidak) [inline guide/english/css/tutorials/transparent-background-images/index.md]: `add :before` -> `add the :before pseudo-selector` (or equivalent)

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@PanSem 
Added some simple typographical changes, please consider the suggestions so that we could merge these

/cc: @RandellDawson

**Comment by RandellDawson** [comment]
Hi @PanSem 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**